README.md

```diff
@@ -132,11 +132,14 @@ dockerfile:
 ```
 
 When `enabled`, go-makefile-maker will generate a `Dockerfile` and a `.dockerignore` file.
+The Dockerfile uses the [Golang base image](https://hub.docker.com/_/golang) to run `make install`, then copies all installed files into a fresh [Alpine base image](https://hub.docker.com/_/alpine).
+The image is provisioned with a dedicated user account (name `appuser`, UID 4200, home directory `/home/appuser`) and user group (name `appgroup`, GID 4200) with stable names and IDs.
```

Comment:
I specifically want to have something that's not just `/` or `/var/empty` because I want to add a shell-rc to the backup-tools image to print a usage message on `kubectl exec` (similar to motd).